docs/project/list-of-diagnostics.md

```diff
@@ -70,6 +70,7 @@ The PR that reveals the implementation of the `<IncludeInternalObsoleteAttribute
 |  __`SYSLIB0018`__ | ReflectionOnly loading is not supported and throws PlatformNotSupportedException. |
 |  __`SYSLIB0019`__ | RuntimeEnvironment members SystemConfigurationFile, GetRuntimeInterfaceAsIntPtr, and GetRuntimeInterfaceAsObject are no longer supported and throw PlatformNotSupportedException. |
 |  __`SYSLIB0020`__ | JsonSerializerOptions.IgnoreNullValues is obsolete. To ignore null values when serializing, set DefaultIgnoreCondition to JsonIgnoreCondition.WhenWritingNull. |
+|  __`SYSLIB0021`__ | Derived cryptographic types are obsolete. |
```

Comment:
When possible, the message should also indicate what should be done to resolve the warning. Is the following accurate, @bartonjs?

```suggestion
|  __`SYSLIB0021`__ | Derived cryptographic types are obsolete. Use the Create method on the underlying type instead. |
```

When this is changed, it will need to be updated in `Obsoletions.cs` and the `ref` sources too.

Reply:
It's technically correct.  I don't know if "underlying type" is the best wording, but I can't come up with an obviously better one.

* "algorithm's abstract class"
* "abstract class representing the algorithm"
* "base type"

Reply:
@gewarren do you have a suggestion for a message better than this?

> Derived cryptographic types are obsolete. Use the Create method on the underlying type instead.

Reply:
I'd suggest "...Use the Create method on the base type instead."

Reply:
> I'd suggest "...Use the Create method on the base type instead."

@jeffhandley apply this?

Reply:
Yes please. Thanks!

_Side note -- I'm sorry about the trouble with the project file; I hope you didn't mind that I pushed commits to fix it. Since that was an existing issue unrelated to your changes, I didn't want to burden you with fixing that up. I'm glad this PR uncovered it though!_

Reply:
> _Side note -- I'm sorry about the trouble with the project file; I hope you didn't mind that I pushed commits to fix it. Since that was an existing issue unrelated to your changes, I didn't want to burden you with fixing that up. I'm glad this PR uncovered it though!_

It's okay, of course I was not against helping with the resolution of problems in the project file! Finished with the documentation :)